contrib/masterfiles-stage/common.sh

```diff
@@ -385,15 +380,22 @@ git_masterstage() {
   # Depends on $GIT_URL, $ROOT, $MASTERDIR, $GIT_REFSPEC
   check_git_installed
   git_setup_local_mirrored_repo "$( dirname "$ROOT" )"
-  if [ "x$1" = "xtrue" ]; then
-    if git_check_is_in_sync "$( dirname "$ROOT" )" "$MASTERDIR" "$GIT_REFSPEC"; then
-      return 1  # in sync => nothing to do
+  git_check_is_in_sync "$local_mirrored_repo" "$MASTERDIR" "$GIT_REFSPEC"
```

Comment:
Might be nice to mention in a comment here that we are abusing the previous function setting a global variable so we know local_mirrored_repo